### PR TITLE
feat(docs): goober documentation

### PR DESCRIPTION
**Pull request by vandreleal**
https://github.com/cristianbote/goober/issues/221

### CODE REVIEW

File: docs/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 gooberjs
```
Comment (cristianbote) [inline docs/LICENSE:3]: Really small nit: I think the project name should be in here without the `js` part. What do you think?

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                {
+                    href: 'https://github.com/cristianbote/goober',
+                    label: 'GitHub',
+                    position: 'right'
+                }
+            ]
+        },
+        footer: {
+            style: 'dark',
+            copyright: `Copyright © ${new Date().getFullYear()} goober. Built with Docusaurus.`
```
Comment (cristianbote) [inline docs/docusaurus.config.js:45]: 😄

**Comment by cristianbote** [comment]
🙌  tremendous job @vandreleal! This looks awesome!

Do you think that the `README` should point to the docs after this? Feels like the information is doubled? Also I kinda think that having the README available with the docs it's nice.

File: docs/LICENSE (hunk 1 above)
Comment (vandreleal) [reply docs/LICENSE]: Good catch! Just removed the `js` part.

**Comment by vandreleal** [comment]
> 🙌 tremendous job @vandreleal! This looks awesome!
> 
> Do you think that the `README` should point to the docs after this? Feels like the information is doubled? Also I kinda think that having the README available with the docs it's nice.

@cristianbote, I think that keeping both of them would be good for those who prefer browsing the `README`, however, there's the small inconvenience of remembering to replicate changes between the `README` and `docs`. The URL to the docs could be pointed using the about repo settings.

**Pull request merged**